src/jdk.jartool/share/classes/sun/tools/jar/resources/jar_de.properties

```diff
@@ -111,7 +111,7 @@ main.help.opt.main.extract=\  -x, --extract              Benannte (oder alle) Da
 main.help.opt.main.describe-module=\  -d, --describe-module      Gibt den Moduldeskriptor oder automatischen Modulnamen aus
```

Comment:
Just because I stumbled over this commit. I think the line should be translated as follows:

`Gibt die Modulbeschreibung oder den automatischen Modulnamen aus`

Reply:
Resolved